Assets/Scripts/Event Manager.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+
+[CreateAssetMenu(fileName = "EventManager", menuName = "Scriptable Objects/EventManager")]
```

Comment:
I really appreciate it when people fully qualify their CreateAssetMenu attribute so this is really really good. Its very easy to make things very messy if this isn't paid attention to. Good job

Reply:
Resolved